### PR TITLE
Copyediting in docs related to HTTP method attributes.

### PR DESCRIPTION
**Pull request by jsampson**

### CODE REVIEW

File: www/docs.md [hunk 1]
```diff
@@ -100,9 +100,12 @@ The core feature of kutty is a set of attributes that allow you to issue AJAX re
...
 * [kt-post](/attributes/kt-post) - Issues a `POST` request to the given URL
-* [kt-put](/attributes/kt-put) - Issues a `PUT` request to the given URL (see [details](#kutty-request-details))
-* [kt-patch](/attributes/kt-patch) - Issues a `PATCH` request to the given URL  (see [details](#kutty-request-details))
-* [kt-delete](/attributes/kt-delete) - Issues a `GET` request to the given URL (see [details](#kutty-request-details))
+* [kt-put](/attributes/kt-put) - Issues a `PUT` request to the given URL
+* [kt-patch](/attributes/kt-patch) - Issues a `PATCH` request to the given URL
+* [kt-delete](/attributes/kt-delete) - Issues a `DELETE` request to the given URL
+
+(Since most browsers only support issuing `GET` and `POST`, a request with one of the other three methods will
+actually be issued as a `POST`, with the `X-HTTP-Method-Override` header set to the desired method.)
```
Comment (jsampson) [inline www/docs.md:108]: There doesn't seem to be a kutty-request-details anchor, so I figured these links could be removed. Then I thought, what's special about these three methods? And I guessed it's the use of X-HTTP-Method-Override, so I borrowed this text from the method attribute pages and generalized it.

**Comment by chg20** [comment]
Thanks Justin!  Great to see you on this project too, kutty is the long term result of a lot of conversations we had back in the day. :)

I am using netlify and deploying straight out of master, so your changes should be live in a minute or two.

**Comment by jsampson** [comment]
It's not live yet, which I'm guessing because this is on dev. :) If I do more pull requests should they be on dev or master?

**Comment by chg20** [comment]
Ha, yeah, if you have a doc fix that can go live against the current site you can do it against master. Sorry should have caught that. :)

Cheers,
Carson

**Pull request merged**